src/Events/Clients/EventsQueueClient.cs

```diff
@@ -2,46 +2,42 @@
 using System.Diagnostics.CodeAnalysis;
 using System.Text;
 using System.Threading.Tasks;
-
+using Altinn.Platform.Events.Clients.Interfaces;
 using Altinn.Platform.Events.Configuration;
 using Altinn.Platform.Events.Models;
-using Altinn.Platform.Events.Services.Interfaces;
-
-using Azure.Storage.Queues;
-
 using Microsoft.Extensions.Options;
 
-namespace Altinn.Platform.Events.Services
+namespace Altinn.Platform.Events.Clients
 {
     /// <summary>
     /// The queue service that handles actions related to the queue storage.
     /// </summary>
     [ExcludeFromCodeCoverage]
-    public class QueueService : IQueueService
+    public class EventsQueueClient : IEventsQueueClient
     {
         private readonly QueueStorageSettings _settings;
 
-        private QueueClient _inboundQueueClient;
+        private Azure.Storage.Queues.QueueClient _inboundQueueClient;
```

Comment:
With the name change of the class, I don't think the full name of the queueClient required.